packages/melos/lib/src/commands/version.dart

```diff
@@ -783,6 +810,29 @@ mixin _VersionMixin on _RunMixin {
       }
     });
   }
+
+  Uri _gitCreateReleaseUrl(
```

Comment:
Why not make this a method of `SupportsManualRelease`?

Reply:
The main reason why I've put it in `version.dart` is that there already were a bunch of similar utility methods and also this way `git_repositories.dart` stays completely free from depending on other files. 

I can still move the method there though if you think that's a better place for it